components/Camera/Camera.tsx

```diff
@@ -0,0 +1,24 @@
+import * as ImagePicker from 'expo-image-picker';
+
+export const openCamera = async () => {
```

Comment:
Question does this function open the camera only? It seems to me that it opens the camera and allows for a picture to be taken. If that's the case I would prefer a different name.